CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+cmake_minimum_required(VERSION 3.20)
+
+project(wxcharts
+        VERSION 0.1.0
+        LANGUAGES CXX)
+
+################
+# config cmake #
+################
+
+set(CMAKE_CXX_STANDARD 11 CACHE STRING "C++ Standard")
+set(CMAKE_CXX_STANDARD_REQUIRED ON CACHE BOOL "")
+set(CMAKE_CXX_EXTENSIONS OFF CACHE BOOL "")
+set(CMAKE_DEBUG_POSTFIX "-d" CACHE STRING "Postfix for debug output")
```

Comment:
I think the wxWidgets core libraries just append "d" and not "-d" so let's try to stay as similar as possible (although personally I prefer "-d" too).

Reply:
OK